serving/build.gradle.kts

```diff
@@ -38,6 +38,7 @@ dependencies {
     testImplementation(libs.testng) {
         exclude(group = "junit", module = "junit")
     }
+    testImplementation("org.mockito:mockito-core:5.11.0")
```

Comment:
-> `testImplementation(libs.mockito.core)`

we already declare the mockito version in gradle/libs.versions.toml (this needs to be updated in the wlm build.gradle.kts too, i can fix that separately)